src/Uno.UI/UI/Xaml/IFrameworkElement.cs

```diff
@@ -98,6 +99,10 @@ public partial interface IFrameworkElement : IUIElement, IDataContextProvider, I
 
 		Windows.UI.Xaml.Media.Brush Background { get; set; }
 
+		//ElementTheme RequestedTheme { get; set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5709307)